spec/system/planning_applications/validating_spec.rb

```diff
@@ -173,6 +172,21 @@
   end
 
   context "Planning application does not transition when expected inputs are not sent" do
+    it "shows an error when invalid documents are present" do
+      create :document, :with_file,
+             planning_application: planning_application,
+             validated: false, invalidated_document_reason: "Missing a lazy Suzan"
+
+      click_link planning_application.reference
+      click_link "Validate application"
+
+      click_button "Validate application"
+      expect(page).to have_content("This application has an invalid document. You cannot validate an application with invalid documents.")
+
+      planning_application.reload
+      expect(planning_application.status).to eql("not_started")
```

Comment:
for the record, you could also do
```rb
       expect(planning_application).to be not_started
```

Reply:
ah thanks - I usually do write it like that but because this was a copy paste I didn't even look at it - will change

Reply:
actually there are about 10 tests in this spec that all use this format so I guess I'll leave it alone for consistency 🤷 unless you'd like me to change all of them?